modules/cco-ccoctl-upgrading.adoc

```diff
@@ -79,17 +73,118 @@ $ oc create namespace <component_namespace>
+=====
+====
 +
-[IMPORTANT]
+.Amazon Web Services (AWS)
+[%collapsible]
 ====
-Refer to the `ccoctl` utility instructions in the installation content for your cloud provider for important platform-specific details about the required arguments and special considerations.
+[source,terminal]
+----
+$ ccoctl aws create-all \// <1>
+  --name=<name> \// <2>
+  --region=<aws_region> \// <3>
+  --credentials-requests-dir=<path_to_directory_with_list_of_credentials_requests>/credrequests \// <4>
+  --output-dir=<path_to_ccoctl_output_dir> \// <5>
+  --create-private-s3-bucket <6>
+----
+<1> To create the AWS resources individually, use the "Creating AWS resources individually" procedure in the "Installing a cluster on AWS with customizations" content. This option might be useful if you need to review the JSON files that the `ccoctl` tool creates before modifying AWS resources, or if the process the `ccoctl` tool uses to create AWS resources automatically does not meet the requirements of your organization.
+<2> Specify the name used to tag any cloud resources that are created for tracking.
+<3> Specify the AWS region in which cloud resources will be created.
+<4> Specify the directory containing the files for the component `CredentialsRequest` objects.
+<5> Optional: Specify the directory in which you want the `ccoctl` utility to create objects. By default, the utility creates objects in the directory in which the commands are run.
+<6> Optional: By default, the `ccoctl` utility stores the OpenID Connect (OIDC) configuration files in a public S3 bucket and uses the S3 URL as the public OIDC endpoint. To store the OIDC configuration in a private S3 bucket that is accessed by the IAM identity provider through a public CloudFront distribution URL instead, use the `--create-private-s3-bucket` parameter.
+====
++
+.Global Microsoft Azure
+[%collapsible]
+====
+[source,terminal]
+----
+$ ccoctl azure create-all \
+  --name=<azure_infra_name> \// <1>
+  --output-dir=<ccoctl_output_dir> \// <2>
+  --region=<azure_region> \// <3>
+  --subscription-id=<azure_subscription_id> \// <4>
+  --credentials-requests-dir=<path_to_directory_with_list_of_credentials_requests>/credrequests \// <5>
+  --dnszone-resource-group-name=<azure_dns_zone_resource_group_name> <6>
+----
+<1> Specify the user-defined name for all created Azure resources used for tracking.
+<2> Optional: Specify the directory in which you want the `ccoctl` utility to create objects. By default, the utility creates objects in the directory in which the commands are run.
+<3> Specify the Azure region in which cloud resources will be created.
+<4> Specify the Azure subscription ID to use.
+<5> Specify the directory containing the files for the component `CredentialsRequest` objects.
+<6> Specify the name of the DNS zone in the resource group to use with Azure DNS.
+====
```

Comment:
As mentioned [here](https://redhat-internal.slack.com/archives/C04TMSTHUHK/p1690570083817259?thread_ts=1690569916.672749&cid=C04TMSTHUHK), if this should be removed until 4.15 lmk and I will just comment it out for now